src/services/authService.js

```diff
@@ -21,42 +24,29 @@ export function ResetPassword(Email) {
 	return http.post(endpointResetPassword, { Email });
 }
 export function CreateReport(data) {
-	const endpointReport = returnConfigItems("endPoints","apiReportUrl");
-	return http.post(endpointReport, data );
+	return http.post(reportItems, data );
 }
-
 export function GetReportByID(ReportID) {
-	const endpointReport =returnConfigItems("endPoints","apiReportUrl");
-	return http.get(endpointReport + ReportID);
+	return http.get(reportItems + ReportID);
 }
-
 export function GetReportByLatLong(X, Y) {
-	const endpointReport = returnConfigItems("endPoints","apiReportUrl");
-	return http.get(endpointReport + X + Y);
-}
 
+	return http.get(reportItems + X + Y);
+}
 export function GetReportByLatLongRadius(X, Y, Radius) {
-	const endpointReport = returnConfigItems("endPoints","apiReportUrl");
-	return http.get(endpointReport + X + Y + Radius);
+	return http.get(reportItems + X + Y + Radius);
 }
-
 export function VerifyAddress(address) {
 	const endpointVerifyAddress = returnConfigItems("endPoints","apiVerifyAddressUrl");
 	return http.get(endpointVerifyAddress + "/" + address);
 }
-
 export function GetContactAddress(id) {
```

Comment:
not blocking, but now that these are one liners, these can be:

```js
export const GetContactAddress = (id) => 
  http.get(contactItems + id + "/addresses");
```

Reply:
fixed